### PR TITLE
Fix language typo in K8s Hello World READMEs

### PR DESCRIPTION
**Pull request by kelsk**

### CODE REVIEW

File: java/java-hello-world/.readmes/intellij/README.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ This "Hello World" sample demonstrates how to deploy a simple "Hello World" appl
 - `skaffold.yaml` - A schema file that defines skaffold configurations ([skaffold.yaml reference](https://skaffold.dev/docs/references/yaml/))
 - `kubernetes-manifests/` - Contains Kubernetes YAML files for the Guestbook services and deployments, including:
 
-  - `hello.deployment.yaml` - deploys a pod with the 'dotnet-hello-world' container image
-  - `hello.service.yaml` - creates a load balancer and exposes the 'dotnet-hello-world' service on an external IP in the cluster
+  - `hello.deployment.yaml` - deploys a pod with the 'java-hello-world' container image
+  - `hello.service.yaml` - creates a load balancer and exposes the 'go-hello-world' service on an external IP in the cluster
```
Comment (ivanporty) [inline java/java-hello-world/.readmes/intellij/README.md:29]: go or java?

Comment (kelsk) [reply java/java-hello-world/.readmes/intellij/README.md]: Copy/Paste strikes again! Fixed!

File: java/java-hello-world/.readmes/theia/README.md [hunk 2]
```diff
@@ -22,8 +22,8 @@ This "Hello World" sample demonstrates how to deploy a simple "Hello World" appl
 - `skaffold.yaml` - A schema file that defines skaffold configurations ([skaffold.yaml reference](https://skaffold.dev/docs/references/yaml/))
 - `kubernetes-manifests/` - Contains Kubernetes YAML files for the Guestbook services and deployments, including:
 
-  - `hello.deployment.yaml` - deploys a pod with the 'dotnet-hello-world' container image
-  - `hello.service.yaml` - creates a load balancer and exposes the 'dotnet-hello-world' service on an external IP in the cluster
+  - `hello.deployment.yaml` - deploys a pod with the 'java-hello-world' container image
+  - `hello.service.yaml` - creates a load balancer and exposes the 'go-hello-world' service on an external IP in the cluster
```
Comment (tejal29) [inline java/java-hello-world/.readmes/theia/README.md]:
```suggestion
  - `hello.service.yaml` - creates a load balancer and exposes the 'java-hello-world' service on an external IP in the cluster
```

**Review by tejal29 — COMMENTED** [review]
replace go to `java` in java examples

**Review by tejal29 — APPROVED** [review]

File: java/java-hello-world/.readmes/theia/README.md (hunk 2 above)
Comment (kelsk) [reply java/java-hello-world/.readmes/theia/README.md]: thanks Tejal!

**Review by ivanporty — APPROVED** [review]

**Pull request merged**